app/Console/Commands/BillingCalculate.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Bill;
+use App\Models\BillHistory;
+use Illuminate\Console\Command;
+use LibreNMS\Util\Number;
+
+require_once base_path() . '/includes/init.php';
```

Comment:
Sorry this needs to be removed

Reply:
Do you mean this line?(so this functions need also converted to laravel?

```
use LibreNMS\Util\Number;
require_once base_path() . '/includes/init.php';
```

This file is required for commands
```
use Illuminate\Console\Command;

```

Reply:
Yes and within there lies the pain ;)

Reply:
Can this line be removed now?